flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -86,6 +86,11 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
             let value = arguments?["value"] as? Any
             setContexts(key: key, value: value, result: result)
 
+        case "removeContexts":
```

Comment:
Call to this code was missing, but i think we don't use it anyway, as we use "loadContexts" instead to merge scope.

Reply:
Good catch, maybe this should make into the main branch together with the fix of the scope sync, I guess this PR wont have any conflicts if you change the target branch, right?